newrelic/resource_newrelic_alert_policy_channel.go

```diff
@@ -83,35 +103,37 @@ func resourceNewRelicAlertPolicyChannelRead(d *schema.ResourceData, meta interfa
 	}
 
 	d.Set("policy_id", policyID)
-	d.Set("channel_id", channelID)
+	d.Set("channel_ids", channelIDs)
```

Comment:
Do we need to set `channel_id` here?

Reply:
This is where the "interesting" logic of whether or not to set `channel_id` or `channel_ids` comes into play.

Reply:
Logic has been updated and is ready for re-review.